chainstate/src/detail/mod.rs

```diff
@@ -191,14 +197,17 @@ impl<S: BlockchainStorage, V: TransactionVerificationStrategy> Chainstate<S, V>
         time_getter: TimeGetter,
     ) -> Self {
         let orphan_blocks = OrphansProxy::new(*chainstate_config.max_orphan_blocks);
+        let events_controller = EventsController::new();
+        let event_broadcast = broadcaster::Broadcaster::new();
```

Comment:
The naming is confusing... we have now two event subscription systems... which I'm not opposed to. But maybe one should reference RPC?

Reply:
Renamed both to make the use of each explicit.